README.md

```diff
@@ -41,6 +41,9 @@ Example usage:
     # TYPE kaiterra_particulate_matter gauge
     kaiterra_particulate_matter{microns="10"} 16
     kaiterra_particulate_matter{microns="2.5"} 15
+    # HELP kaiterra_rtvoc Total Volatile Organic Compounds (TVOC) in ppb
+    # TYPE kaiterra_rtvoc gauge
+    kaiterra_rtvoc 486
```

Comment:
What's the "R" stand for?

Reply:
I'm not sure, it is part of the Parameter Codes from https://www.kaiterra.com/dev/#header-sensor-reading-data-format

rtvoc	| Total Volatile Organic Compounds (TVOC) |	ppb

Instead, I like the idea of renaming the variable and spelling it out. I'll move forward with that, thanks